students/BrandonHenson/Lesson6/test_mailroom4.py

```diff
@@ -0,0 +1,34 @@
+# Brandon Henson
+# 4/27/18
+# Lesson 06 mailroom_4.py
+# Testing
+
+import unittest
+import Mailroom_4
+import os
+
+class MailroomTest(unittest.TestCase):
+
+    def setUp(self):
+        Mailroom_4.donor_history = {'Brandon Henson': [1005.49, 3116.72, 5200]}
+    def test_donor_history(self):
+        self.assertEqual(Mailroom_4.donor_history,{'Brandon Henson': [1005.49, 3116.72, 5200]})
+        Mailroom_4.donor_history['test']=45
+        self.assertEqual(Mailroom_4.donor_history,{'Brandon Henson': [1005.49, 3116.72, 5200], 'test': 45})
```

Comment:
this test isn't testing your code, it's really just testing dictionary functionality which does not need to be tested.